library/responsive-images.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Configure responsive images sizes
+ *
+ * @package WordPress
+ * @subpackage FoundationPress
+ * @since FoundationPress 2.2.0
+ */
+
+ // Add additional image sizes
+ add_image_size( 'fp-small', 640 );
+ add_image_size( 'fp-medium', 1024 );
+ add_image_size( 'fp-large', 1200 );
+ add_image_size( 'fp-xlarge', 1440 );
+
+ // Register the new image sizes for use in the add media modal in wp-admin
+ add_filter( 'image_size_names_choose', 'wpshout_custom_sizes' );
+ function wpshout_custom_sizes( $sizes ) {
```

Comment:
I assume the code comes from [this WPShout article](http://wpshout.com/adding-using-custom-image-sizes-wordpress-guide-best-thing-ever/) but shouldn't custom function in the theme start with the same `foundationpress_` rather than using `wpshout_` from the article?